### ISSUE TITLE
Self-destruct messages + Delete all messages by one button

### ISSUE THREAD

**Issue by amaroc** [vector-im/riot-android#2254]
Dear guys and developers. Please stop disqus about self-destruct messages and making a new vich bicycle. In Telegram you can try this function. And this function must-have. if you dont want do this function just give "One button" to delete and clear all chat messages in 1:1 chat. Without this all your e2e senselessly.

And another good option for your chat is keep data  (messages and media files) in crypto containers wich can work only after entering password.

**Comment by takiainen** [comment]
I'd like to see these features as well.

**Comment by moralrebuild** [comment]
a mainstream critical feature that is missing for a long time

**Comment by Thatoo** [comment]
duplicate of https://github.com/vector-im/riot-android/issues/1246
